Web/starter_project_vanguard/components/success-story/SuccessDescription.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Story } from '@/types/success-story'
+
+
+const SuccessDescription: React.FC<Story> = ({ heading, paragraph }) => {
```

Comment:
let's have a props definition on top of each component

Reply:
```
interface SuccessDescriptionProps {
  story: Story;
}
```

when using it it would look like
```
const SuccessDescription: React.FC<SuccessDescriptionProps> = ({ story: { heading, paragraph } }) => {
```